developer_setup/classic_ui_split.md

```diff
@@ -16,19 +16,11 @@ But if you do..
 1. `git remote add upstream git@github.com:ManageIQ/manageiq-ui-classic.git`
 1. `ln -s ../../manageiq spec/`
 1. `cd ../manageiq`
-1. `echo 'unless dependencies.detect { |d| d.name == "manageiq-ui-classic" }' >> Gemfile.dev.rb`
-1. `echo '  gem "manageiq-ui-classic", :path => "'$(cd ../manageiq-ui-classic/; /bin/pwd)'"' >> Gemfile.dev.rb` (because you really need an absolute path there..)
-1. `echo 'end' >> Gemfile.dev.rb`
+1. `echo "override_gem 'manageiq-ui-classic', :path => File.expand_path('../manageiq-providers-amazon', __dir__) >> Gemfile.dev.rb`
```

Comment:
Well, probably not `../manageiq-providers-amazon` ;)

(And this needs testing, I'm not that sure that the absolute path is no longer needed..)